tests/ContainerTest.php

```diff
@@ -148,6 +148,24 @@ public function testNewInstanceWithOverride()
         $this->assertSame($expect, $actual);
     }
 
+    /**
+     * @requires PHP 5.6
+     */
+    public function testNewInstanceWithVariadic()
+    {
+        if (defined('HHVM_VERSION')) {
```

Comment:
May be good to add for php 5.5 ? Same for below. And thank you for the fast response.

Reply:
The @requires has the same effect, and is generally the preferred/recommended approach - but happy to add a code check too if you prefer?

Reply:
I think HHVM and PHP differs. Not sure what you were pointing though. Can you give me a link to the explanation where the `preferred/recommended` one ?